skbeam/core/accumulators/binned_statistic.py

```diff
@@ -468,7 +468,7 @@ def __init__(self, shape, bins=10, range=None,
                 represented by function([]), or NaN if this returns an error.
         """
         if origin is None:
-            origin = shape[0]//2, shape[1]//2
+            origin = shape[0]/2., shape[1]/2.
```

Comment:
I believe it's only used to setup the grid, which I believe doesn't require integers.  But you're probably right:  the doc string should be updated.